bolt-modules/out/lib/puppet/functions/out/verbose.rb

```diff
@@ -0,0 +1,75 @@
+# frozen_string_literal: true
+
+# Output a message for the user when running in verbose mode.
+#
+# This will print a message to stdout when using the human output format,
+# and print to stderr when using the json output format.
+#
+# > **Note:** Not available in apply block
+Puppet::Functions.create_function(:'out::verbose') do
+  # @param message The message to output.
+  # @example Print a message
+  #   out::verbose('Something went wrong')
+  dispatch :output_message do
```

Comment:
```suggestion
  dispatch :output_verbose do
```
It doesn't really matter what this is called, but I think having it be something different from the `out::message` dispatch is more readable.

Reply:
I would say I understand what the dispatch does about 75%. So a follow up to this is, does this mean I should also change `def output_message(message)` on line 18 to `def output_verbose(message)` ?

Reply:
Oo yes - I think you probably found the answer to this 😅 but yeah, the dispatch will call the Ruby function with the same name.